README.md

```diff
@@ -59,8 +59,8 @@ man f3d # Linux only
 
 F3D needs you help!
 
-If you can, please consider sponsoring F3D, even a small amount, it would help us offsetting the maintenance cost
-as well as considering adding support for devices we do not own yet. Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).
+If you can, please consider sponsoring F3D. Even a small donation would help us offsetting the recurring maintenance costs.
```

Comment:
```suggestion
If you can, please consider sponsoring F3D. Even a small donation would help us offset the recurring maintenance costs.
```
I probably could have proof read my own suggestion better the first time around...